tests/test_gbdesAstrometricFit.py

```diff
@@ -1058,6 +1061,114 @@ def test_inputCameraModel(self):
                     self.assertAlmostEqual(np.std(dDec), 0)
 
 
+class TestApparentMotion(lsst.utils.tests.TestCase):
+    """This class simulates an array of objects with random proper motions and
+    parallaxes and compares the results of
+    `lsst.drp.tasks.gbdesAstrometricFit.calculate_apparent_motion` against the
+    results calculated by astropy."""
+
+    @classmethod
+    def setUpClass(cls):
+        np.random.seed(12345)
+
+        cls.RAs = np.random.rand(10) + 150.0
+        cls.Decs = np.random.rand(10) + 2.5
+
+        cls.pmRAs = np.random.random(10) * 10
+        cls.pmDecs = np.random.random(10) * 10
```

Comment:
You've got a mix of camelCase and snake_case in this file (e.g. `ra_correction` below). We're moving to snake_case in general, so it might be worth doing that here.
```suggestion
        cls.pm_ra = np.random.random(10) * 10
        cls.pm_dec = np.random.random(10) * 10
```

Reply:
I've gone through and made the capitalization consistent with the rest of the `gbdesAstrometricFit` module, which is snake_case for function names and camelCase otherwise.